python/cuml/ensemble/randomforest_common.pyx

```diff
@@ -285,9 +285,15 @@ class BaseRandomForestModel(Base):
             self.num_classes = len(self.classes_)
             for i in range(self.num_classes):
                 if i not in self.classes_:
-                    raise ValueError("The labels need "
-                                     "to be consecutive values from "
-                                     "0 to the number of unique label values")
+                    self.classes_unorder = cp.unique(y_m).tolist()
```

Comment:
We should be reusing existing primitives where at all possible and using the make_monotonic primitive to do this. That allows us to optimize this specific operation once and have it benefit all uses.